pipelines/rj_cor__precipitacao_alertario/flow.py

```
@@ -70,14 +71,16 @@ def rj_cor__precipitacao_alertario(
         - Dados pluviométricos incluem acumulados de chuva em diversos intervalos
         - Dados meteorológicos incluem temperatura, umidade, pressão e vento
     """
+    rename_current_flow_run_task
```

Ruff (0.15.17)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)